src/modules/catalog.js

```diff
@@ -1166,7 +1166,7 @@ class Catalog {
    * @param {object} [networkParameters] - Parameters relevant to the network request
    * @param {number} [networkParameters.timeout] - Request timeout (in milliseconds)
    * @returns {Promise}
-   * @see https://docs.constructor.io/rest-api.html
+   * @see https://docs.constructor.io/rest_api/
```

Comment:
Can probably change the synonym ones to point to `https://docs.constructor.io/rest_api/synonyms/`